packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -148,6 +154,9 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
 
   public componentWillUnmount() {
     delete _allInstances[this._id];
+
+    // Dispose all events.
+    this._disposables.forEach(d => d());
```

Comment:
I wonder if we could make a thing `Disposables` helper that wraps the Function array and provides some basic helpers, like `dispose()`. Somewhat minor, but I think it would help ensure devs use it properly.

Reply:
I added the "extendComponent", so it's possible...

Reply:
but not in this pr.